src/main/java/frc/team2412/robot/subsystems/LimelightSubsystem.java

```diff
@@ -0,0 +1,196 @@
+package frc.team2412.robot.subsystems;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.team2412.robot.commands.drivebase.DriveCommand;
+import java.util.function.DoubleSupplier;
+
+public class LimelightSubsystem extends SubsystemBase {
+
+	// CONSTANTS
+
+	private static final PIDController TRANSLATION_PID = new PIDController(10.0, 0, 0);
+	private static final PIDController ROTATION_PID = new PIDController(8.0, 0, 0);
+
+	// meters?
+	public static final double CAMERA_MOUNT_HEIGHT = 0.1143;
+	public static final double CAMERA_ANGLE_OFFSET = 0;
```

Comment:
Is this accurate? Looking at the physical setup on this year's robot, the camera is at an angle.

Reply:
Distance finder no longer requires these values, removed